appveyor.yml

```diff
@@ -1,4 +1,4 @@
-version: 11.0.{build}
+version: 11.1.{build}

```

Comment:
Merk op dat je nu in 2 files moet editen omdat de versies door appveyor geregeld worden: onnodige complexiteit.

Reply:
oh, zie andere PR: ja, dat is mijn voorstel.

Reply:
(Helemaal fijn zou zijn als we zelfs de push niet automatiseren, maar tegelijkertijd het wel mogelijk maken dat iedereen dat met een druk op de knop kan of ev.t dat het via een github tag kan of alleen als de versie geupdate is of whatever, maar dat is vermoedelijk complexer)

Reply:
Het kan waarschijnlijk allemaal. Er zit zelfs een deploy knopje in AppVeyor (welke nu nog niet werkt, maar dat valt vast te configureren).